docs/bootstrap.md

```diff
@@ -1,35 +1,56 @@
-# Bootstrapping a Kubernetes cluster
+# Bootstrapping Dragonfly on Digital Ocean
 
-## Create namespaces
+## Prerequisites
 
-```bash
-kubectl apply -f kubernetes\manifests\cert-manager\namespace.yaml
-kubectl apply -f kubernetes\manifests\discord\namespace.yaml
-kubectl apply -f kubernetes\manifests\dragonfly\namespace.yaml
+You will need:
+
+* [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/) configured with a...
+* [Digital Ocean Personal Access Token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) with at least the following scopes:
+  * kubernetes (2): create, delete
+  * If replacing an existing cluster:
+    * load\_balancer (1): delete

```

Comment:
`question`: We'll need this as well, right?
```md
kubernetes: access_cluster
```

Reply:
That scope only controls access to these two endpoints, which I'm pretty sure we don't (technically) need:
* https://docs.digitalocean.com/reference/api/api-reference/#operation/kubernetes_get_kubeconfig
* https://docs.digitalocean.com/reference/api/api-reference/#operation/kubernetes_get_credentials

Reply:
Okay, we can get those from the UI, I think.